### ISSUE TITLE
signal.mjs has some invalid @returs tags

### ISSUE THREAD

**Issue by giohappy** [tidalcycles/strudel#167]
Try to build execute jsdoc the following errors are thrown:

```
ERROR: The @returns tag requires a value. File: signal.mjs, line: 136
ERROR: The @returns tag requires a value. File: signal.mjs, line: 143
ERROR: The @returns tag requires a value. File: signal.mjs, line: 147
ERROR: The @returns tag requires a value. File: signal.mjs, line: 154
```

**Comment by felixroos** [comment]
Yep, the returns are missing, do you want to fix it?

**Comment by felixroos** [comment]
fixed with #169 thanks for reporting!

**Comment by giohappy** [comment]
thanks @felixroos. Sorry, I was away all the day and I couldn't send a PR.
